assets/airports/cyow.json

```diff
@@ -0,0 +1,1279 @@
+        ],
+        "ils": [false, true]
+    }, {
+        "name": ["07", "25"],
+        "end": [
+            ["N45d18m47.80", "W075d40m16.80", "369ft"],
+            ["N45d19m30.20", "W075d38m42.40", "362ft"]
+        ],
+        "ils": [true, true]
+    }, {
+        "name": ["04", "22"],
+        "end": [
+            ["N45d19m36.00", "W075d41m20.50", "347ft"],
+            ["N45d20m03.10", "W075d41m02.30", "335ft"]
+        ],
+        "ils": [true, true]
+    }],
+    "sids": {
+        "CYOW4": {
+            "icao": "CYOW4",
+            "name": "Ottawa Four",
+            "rwy": {
+                "04": ["#039"],
+                "07": ["#071"],
+                "14": ["#140"],
+                "22": ["#219"],
+                "25": ["#251"],
+                "32": ["#320"]
+            },
+            "exitPoints": {
+                "APLOV": ["APLOV"],
+                "AVVON": ["AVVON"],
+                "IKLAX": ["IKLAX"],
+                "KODEX": ["KODEX"],
+                "LETAK": ["LETAK"],
+                "OLIGO": ["OLIGO"],
+                "RADEN": ["RADEN"],
+                "TUKIR": ["TUKIR"]
+            },
+            "draw": [
+                ["APLOV*"],
+                ["AVVON*"],
+                ["IKLAX*"],
+                ["KODEX*"],
+                ["LETAK*"],
+                ["OLIGO*"],
+                ["RADEN*"],
+                ["TUKIR*"]
+            ]
+        }
+    },
+    "stars": {
+        "CAPTL4": {
+            "icao": "CAPTL4",
+            "name": "Capital Four",
+            "entryPoints": {
+                "MIVOK": ["MIVOK"],
+                "ELSUB": ["ELSUB"]
+            },
+            "body": [["KANIK", "A230"]],
+            "rwy": {
+                "07": [["LANRK", "A100-"], ["VISOL", "A30"], "#071"],
+                "14": [["LANRK", "A160-"], ["VISOL", "A50"], ["SEBEG", "A50"], ["MEGNO", "A35"], "OMTUN", "#320"],
+                "25": [["LANRK", "A160-"], ["VISOL", "A50"], "ULAGU", ["BOREG", "A50|S230"], ["ADVOS", "A30|S210"], "KAREB", "#084"],
+                "32": [["LANRK", "A160-"], ["VISOL", "A50"], "ULAGU", ["DEMKO", "A50|S210"], ["VOLAG", "A30"], "DUKTI", "#140"],
+                "22": []
+            },
+            "draw": [
+                ["MIVOK*", "KANIK"],
+                ["ELSUB*", "KANIK", "LANRK", "VISOL"],
+                ["VISOL", "ULAGU", "BOREG", "ADVOS", "KAREB"],
+                ["ULAGU", "DEMKO", "VOLAG", "DUKTI"],
+                ["VISOL", "SEBEG", "MEGNO", "OMTUN"]
+            ]
+        },
+        "DEANS8": {
+            "icao": "DEANS8",
+            "name": "Deans Eight",
+            "entryPoints": {
+                "ART": ["ART"],
+                "BUGSY": ["BUGSY", ["SAVAL", "A50+"], ["TAPVO", "A180-|A40+"]]
+            },
+            "body": [],
+            "rwy": {
+                "07": [["CYRIL", "A120"], ["CAVUR", "A75|S230-"], ["AMAGI", "A50|S210-"], ["NOVUX", "A50"], ["TILOG", "A30"], "EBNAP", "#247"],
```

Comment:
CYRIL should be `["CYRIL", "A120-"]`
CAVUR should be `["CAVUR", "A75+|S230-"]`
TILOG should be `["TILOG", "A30+"]`